eng/build.ps1

```diff
@@ -63,6 +63,33 @@ param (
 Set-StrictMode -version 2.0
 $ErrorActionPreference = "Stop"
 
+[Reflection.Assembly]::LoadWithPartialName("System.Drawing")
+[Reflection.Assembly]::LoadWithPartialName("System.Windows.Forms")
```

Comment:
❕ These need semicolons to avoid printing lines to the console when the script runs.

Reply:
➡️ Now using `Add-Type`